guestbook.module

```diff
@@ -0,0 +1,177 @@
+<?php
+module_load_include('inc', 'guestbook', 'include/guestbook_db');
+module_load_include('inc', 'guestbook', 'include/guestbook_core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+module_load_include('inc', 'guestbook', 'include/guestbook_weather');
+
+/**
+* @file
+* Build page, form for guestbook and weather block.
+*/
+
+/**
+* Implements hook_menu(). 
+*/
+ function guestbook_menu() {
+   $items = array();
+   // Main page.
+   $items['guestbook-page'] = array(
+     'title' => 'Guestbook',
+     'page callback'   => 'drupal_get_form',
+     'page arguments'  => array('guestbook_page_form'),
+     'access callback' => TRUE,
+  );
+   // Edit message.
+   $items['guestbook-page/%mid/edit'] = array(
+     'title'            => 'Guestbook',
+     'page callback'    => 'drupal_get_form',
+     'page arguments'   => array('guestbook_page_form', 1),
+     'access callback'  => TRUE,
+     'type'             => MENU_CALLBACK,
+
+   );
+   // Delete message.
+   $items['guestbook-page/%mid/delete'] = array(
+     'title'            => 'Delete message',
+     'page callback'    => 'guestbook_message_delete',
+     'page arguments'   => array(1),
+     'access callback'  => TRUE,
+     'type'             => MENU_CALLBACK,
+   );
+   return $items;
+}
+
+/**
+ * Implements hook_form.
+ *
+ * @param $value
+ *   Implements hook_load
+ *   contains array('id', 'uid', 'sid', 'date', 'message').
+ * @return
+ *   Return form.
+ */
+function guestbook_page_form($form, $form_state, $value= NULL) {
+  global $user;
+  // Text for editing.
+  $text_edit = NULL;
+
+  // Get html list of messages.
+  $text = guestbook_html();
+
+  $form = [];
+
+  if (isset($value['id'])) {
+    // Get tex for editing.
+    $text_edit = $value['message'];
+    $form['infoById'] = array('#type' => 'value', '#value' => $value);
+  }
+
+  $form['#attached']['css'][] = drupal_get_path('module', 'guestbook') . '/css/guestbook.css';
+
+  $form['message_window'] = [
+    // Show messages if it exists.
+    '#markup' => $text,
+  ];
+
+  // Show area for input text if user is logged.
+  if ($user->uid) {
+    $form['message'] = [
+      '#type' => 'textarea',
+      '#title' => t('message'),
+      '#size' => 1000,
+      '#default_value' => $text_edit,
+    ];
+
+    $form['submit'] = [
+      '#type' => 'submit',
+      '#attributes' => array('class' => array('guestbook-button')),
+      '#value' => isset($value['id']) ? t('Edit') : t('Save'),
+    ];
+
+    // Show or hide the cancel button.
+    if (isset($value['id'])) {
+      $form['cancel_button'] = [
+        '#type' => 'link',
+        '#title' => t('Cancel'),
+        '#href' => 'guestbook-page',
+        '#options' => array('query' => drupal_get_query_parameters()),
+        '#attributes' => array('class' => array('guestbook-button')),
+
+      ];
+    }
+  }
+  return $form;
+}
+
+/**
+ * Validate guestbook_page_form.
+*/
+function guestbook_page_form_validate($form, &$form_state) {
```

Comment:
Move to guestbook.form.inc and move all code related to form to this file (delete callback, validation and so on)